docs/functions/vRealize Log Insight/Set-VCFvRLI.md

```diff
@@ -0,0 +1,58 @@
+# Set-VCFvRLI
```

Comment:
Not seeing the Set-VCFvRLI in the main file ?

Reply:
Added in a later commit